packages/docs/docs/changelog.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 0.8.2
+## 0.8.3
+
+- Add OpenTelemetry integration for AI.JSX render tracing, which can be enabled by setting the `AIJSX_ENABLE_OPENTELEMETRY` environment variable.
```

Comment:
Would you mind letting https://github.com/fixie-ai/ai-jsx/pull/223 land first? I've already had to deal with a few conflicts in that PR. :smile: